Modules/FindCUDA.cmake

```diff
@@ -634,6 +682,9 @@ find_program(CUDA_NVCC_EXECUTABLE
 find_program(CUDA_NVCC_EXECUTABLE nvcc)
 mark_as_advanced(CUDA_NVCC_EXECUTABLE)
 
+#MESSAGE("CUDA_NVCC_EXECUTABLE: ${CUDA_NVCC_EXECUTABLE} CUDA_TOOLKIT_ROOT_DIR: ${CUDA_TOOLKIT_ROOT_DIR} CUDA_TOOLKIT_TARGET_DIR: ${CUDA_TOOLKIT_TARGET_DIR}")
```

Comment:
probably I should put those under if (verbose) - having those when needed saves a lot of typing.